config/set/code-quality.php

```diff
@@ -126,6 +127,7 @@
         SimplifyIfNotNullReturnRector::class,
         SimplifyIfReturnBoolRector::class,
         SimplifyUselessVariableRector::class,
+        SimplifyUselessLastVariableAssignRector::class,
```

Comment:
Please revert this, I or @TomasVotruba can add it later and ensure that auto-commit Rectify (which can't runnable on fork) with added rule to the config set doesn't cause a regression on existing codebase.